components/dialogs/PersistentDialog.vue

```diff
@@ -26,6 +26,33 @@
                 </v-card-actions>
             </v-card>
         </v-dialog>
+
+        <v-dialog v-model="dialog" persistent no-click-animation=true max-width="290">
```

Comment:
Could you tell me please what property is responsible for that?

Reply:
If you use no-click-animation there is no animation when you click outside an open persistent dialog, if you open this dialog the first time after the page reloading. Applicable for persistent dialogs only